src/components/Dropdown.js

```diff
@@ -0,0 +1,25 @@
+// DropDown.js
+import { useState } from "react";
+
+const Dropdown= ({breeds, getCatData}) => {
+const [ userChoice, setUserChoice] = useState("placeholder");
+const userInputHandler = (e) => {
+    setUserChoice(e.target.value);
+}
+
+return (
+<form onSubmit={(e) => {getCatData(e, userChoice)}}> 
+   
+  <select onChange = {userInputHandler} className="dropdown"> 
+    <option value = "placeholder" key="placeholder">Please select a cat.</option>  
```

Comment:
WOW! Your getting the values for the drop down straight from the API . I really like that very smart!